ruby-engine/ext/extconf.rb

```diff
@@ -0,0 +1,2 @@
+require 'mkmf'
+create_makefile('unleash-engine/unleash-engine')
```

Comment:
Snuck in because I need a useful build for ruby and this doesn't work without this. Plz ignore, soz